public/app/core/utils/query.ts

```diff
@@ -1,14 +1,4 @@
-import { DataQuery, DataSourceRef } from '@grafana/data';
-
-// @deprecated use the `getNextRefId` function from grafana/data instead
-export const getNextRefIdChar = (queries: DataQuery[]): string => {
```

Comment:
In case anyone wants to check, this is a 1-1 with grafana/data 

https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/query/refId.ts#L6